src/app/app.component.ts

```diff
@@ -10,26 +11,28 @@ import { ButtonLabels } from "./shared/constants/button-labels";
 })
 export class AppComponent {
   title = "courses-app";
+  isAuthorized: boolean = false;
+  isAdmin!: boolean;
+  userName: string | null = "";
 
   //  text for buttons
   btnTextLogin: string = ButtonLabels.login;
   btnTextLogout: string = ButtonLabels.logout;
-  btnTextAddNewCourse: string = ButtonLabels.addNewCourse;
 
-  // info page text
-  infoTitle = "Your List Is Empty";
-  infoText = "Please use 'Add New Course' button to add your first course";
+  constructor(
+    public authService: AuthService,
+    private userStoreService: UserStoreService
+  ) {}
 
-  // Mocked Course
-  mockedCourse = mockedCoursesList[0];
-  mockedCoursesList = mockedCoursesList;
-  showCourse(courseId: string) {
-    console.log("Clicked on Show Course with ID:", courseId);
+  ngOnInit() {
```

Comment:
Instead of dealing with isAdmin and name properties of the user separately, I suggest that you handle a single User object in the user service, store. More on this on the corresponding file.